plugins/inputs/azure_monitor/README.md

```diff
@@ -77,6 +82,8 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   client_secret = "<<CLIENT_SECRET>>"
   # can be found under Azure Active Directory->Properties
   tenant_id = "<<TENANT_ID>>"
+  # Define the optional Azure cloud option e.g. AzureChina, AzureGovernment or AzurePublic | DEFAULT AzurePublic
+  cloud_option = "<<Cloud_Option>>"
```

Comment:
Since there is a default (e.g. AzurePublic), can we please put that here and comment out the config field?

Reply:
I'll adapt it.